src/etc/tidy.py

```diff
@@ -24,6 +24,7 @@
 
 interesting_files = ['.rs', '.py', '.js', '.sh', '.c', '.h']
 uninteresting_files = ['miniz.c', 'jquery', 'rust_android_dummy']
+stable_whitelist = []
```

Comment:
Weird, Travis says that `make tidy` passes. Have you tried purposefully omitting an `#[unstable]` annotation to see if the code catches that?